app/components/panel_component/panel.scss

```diff
@@ -28,6 +28,16 @@
       }
     }
   }
+
+  .search-controls__alternative {
```

Comment:
I think we should use double hyphen here. We're following this convention: https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/

Reply:
yes correct @david-mears-dfe - alternative is a modifier strictly speaking